packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -1608,10 +1562,25 @@ class SvgAttributes {
   /// The `mix-blend-mode` attribute.
   final BlendMode? blendMode;
 
+  /// The `font-family` attribute, as a raw string.
+  final String? fontFamily;
+
+  /// The `font-weight` attribute, as a raw string.
+  final String? fontWeight;
+
+  /// The `font-size` attribute, as a raw string.
+  final String? fontSize;
```

Comment:
I think it'd probably be good to have these parsed into the right enum/value types, like we're doing for `blendMode` etc.

Reply:
done